src/main/java/reposense/git/GitDiff.java

```diff
@@ -18,4 +23,14 @@ public static String diffCommit(String root, String lastCommitHash) {
         Path rootPath = Paths.get(root);
         return runCommand(rootPath, "git diff -U0 " + lastCommitHash);
     }
+
+    /**
+     * Returns a list of committed files with the corresponding number of lines added and deleted in the repo
+     * {@code repoRoot}.
+     */
+    public static List<String> getModifiedFiles(Path repoRoot) {
+        String diffCommand = String.format("git diff --numstat %s %s", EMPTY_TREE_HASH, CHECKED_OUT_COMMIT_REFERENCE);
+        String diffResult = runCommand(repoRoot.toAbsolutePath(), diffCommand);
+        return new ArrayList<>(Arrays.asList(diffResult.split("\n")));
```

Comment:
Are we modifying the returned list?

If not, can just `return Arrays.asList(diffResult.split("\n"));`?